### PR TITLE
Fix typos and spelling errors

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview
Fix various typos and spelling

## Functional Changes
None

## Manual Testing Performed
None, avoided changing typos that might be used in delegates

### CODE REVIEW

File: http-server/src/test/java/org/triplea/server/moderator/toolbox/api/key/InvalidKeyCacheTest.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 @ExtendWith(MockitoExtension.class)
 class InvalidKeyCacheTest {
 
-  private static final String IP_ADDRESS_1 = "Grogs are the cannibals of the big urchin.";
+  private static final String IP_ADDRESS_1 = "Grog are the cannibals of the big urchin.";
```
Comment (RoiEXLab) [inline http-server/src/test/java/org/triplea/server/moderator/toolbox/api/key/InvalidKeyCacheTest.java]: 'Grog are'?

Comment (DanVanAtta) [reply http-server/src/test/java/org/triplea/server/moderator/toolbox/api/key/InvalidKeyCacheTest.java]: ya, the spell checker did not like the 'grogs' word, minimal update here even if the grammar is now very weird.

File: game-core/src/main/java/games/strategy/net/IpFinder.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
 /**
  * nekromancer@users.sourceforge.net Utility class for finding the local ip address of a machine
  * with multiple network interfaces. This class will discard any InetAddress whose
- * isLoobackAddresS() returns true. It will also discard any InetAddress whose isLinkLocalAddress()
+ * isLoopbackAddresS() returns true. It will also discard any InetAddress whose isLinkLocalAddress()
```
[the remark this answers is not in the document]
Comment (DanVanAtta) [reply game-core/src/main/java/games/strategy/net/IpFinder.java]: Only spotting the one, where is second?

[the remark this answers is not in the document]
Comment (RoiEXLab) [reply game-core/src/main/java/games/strategy/net/IpFinder.java]: Yeah I meant to say "there were", so you spotted the second one already

**Pull request merged**